### PR TITLE
FR-17412 - Upgrade frontegg native SDKs

### PR DESCRIPTION
**Pull request by frontegg-david**

### CODE REVIEW

File: lib/src/frontegg_method_channel.dart [hunk 1]
```diff
@@ -3,55 +3,95 @@ import "package:flutter/services.dart";
...
   @override
-  Future<void> directLoginAction(String type, String data) => methodChannel.invokeMethod(
+  Future<void> directLoginAction(String type, String data,
+      {bool ephemeralSession = true}) =>
+      methodChannel.invokeMethod(
         directLoginActionMethodName,
         {
           "type": type,
           "data": data,
+          "ephemeralSession": ephemeralSession,
```
Comment (alex-minaiev-frontegg) [inline lib/src/frontegg_method_channel.dart:73]: Do we need to use the `ephemeralSession` param on iOS side(ios/Classes/FronteggMethodCallHandler.swift.directLoginAction)?

Comment (alex-minaiev-frontegg) [reply lib/src/frontegg_method_channel.dart:73]: Also, add the `ephemeralSession` to lib/src/frontegg_flutter.dart.directLoginAction so the customer developer can use this param. And make this params by default([bool ephemeralSession = true] or {bool ephemeralSession = true} )

Comment (frontegg-david) [reply lib/src/frontegg_method_channel.dart:73]: It's already have default value

Comment (alex-minaiev-frontegg) [reply lib/src/frontegg_method_channel.dart:73]: Yep.
We have a default value for `FronteggMethodChannel`. But users gonna use FronteggFlutter().directLoginAction, not FronteggMethodChannel().directLoginAction. 
If we don't wanna users to set up `ephemeralSession` for `directLoginAction` it is ok. But if we wanna users set up `ephemeralSession` we must add `ephemeralSession` to lib/src/frontegg_flutter.dart and set it on FronteggMethodCallHandler.swift.directLoginAction

Comment (frontegg-david) [reply lib/src/frontegg_method_channel.dart:72]: fixed

**Review by alex-minaiev-frontegg — APPROVED** [review]

**Pull request merged**